centreon/www/include/options/accessLists/menusACL/DB-Func.php

```
@@ -562,28 +592,40 @@ function updateGroups($aclId = null)
+        );
+        $prepareDelete->bindValue(':acl_id', $aclId, PDO::PARAM_INT);
+        $prepareDelete->execute();
 
-    if ($prepareDelete->execute()) {
         $submitedValues = $form->getSubmitValue('acl_groups');
         if (isset($submitedValues)) {
+            $insertStmt = $pearDB->prepare(
+                'INSERT INTO acl_group_topology_relations (acl_topology_id, acl_group_id)
+                VALUES (:aclId, :value)'
+            );
             foreach ($submitedValues as $key => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)